lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -52,7 +55,18 @@ const ItineraryWrapper = styled.div.attrs((props) => {
   padding: 0;
 `
 
+// invisible header rendered for screen readers and a11y technologies
+const InvisibleHeader = styled.h3`
+  width: 0;
+  height: 0;
+  overflow: hidden;
+  //place it in an unused grid cell so it doesn't add a row
```

Comment:
Not sure how I feel about this method of hiding the element -- will leave it up to the second reviewer to decide

Reply:
This is fine. Eventually we'll need to combine all the "visually hidden" CSS into a single place.